composer.json

```diff
@@ -20,8 +20,11 @@
             "role": "Developer"
         }
     ],
+    "minimum-stability": "dev",
     "require": {
-        "php": "^7.4|^8.0"
+        "php": "^7.4|^8.0",
+        "sentry/sentry": "*",
```

Comment:
Please don't require any version, but specify what versions are supported!

https://thephp.cc/articles/the-death-star-version-constraint